geth-with-protocol/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN echo "" > /geth/password.txt
 RUN geth --datadir $gethRoot init /geth/genesis.json
 RUN cp /geth/keys/* $gethRoot/keystore/
 
-FROM node:10-alpine as builder-node
+FROM node:14.4.0-alpine as builder-node
```

Comment:
Was this `node` version chosen for compatibility? It looks like the latest v14 release is:

https://github.com/nodejs/node/releases/tag/v14.19.1

Reply:
Sounds good, we may want to change this to `14-alpine` to reduce maintenance